src/api.js

```diff
@@ -0,0 +1,29 @@
+export { getUserId, getUserFolder };
+
+async function getUserId() {
+  const response = await fetch(
+    "https://bootcamp-api.codeit.kr/api/sample/user",
+    {
+      method: "GET",
+      headers: {
+        accept: "*/*",
+      },
+    }
+  );
+  const body = await response.json();
+  return body;
```

Comment:
[중요도: 중간]
```suggestion
  return await response.json();
```
으로 해보는건 어떨까요?

Reply:
https://github.com/codeit-bootcamp-frontend/3-Weekly-Mission/pull/165/commits/763f7eb933a47f07d641e59a6268fa9da048cc96

수정했습니다!
군더더기가 붙어있었군요